Snowflake.Data.Tests/IntegrationTests/ConnectionPoolCommonIT.cs

```diff
@@ -64,34 +64,6 @@ public void TestBasicConnectionPool()
             Assert.AreEqual(1, SnowflakeDbConnectionPool.GetPool(ConnectionString).GetCurrentPoolSize());
         }
 
-        [Test]
```

Comment:
we can't keep it together because:
- in the old pool the only way to change expiration timeout was to use `SnowflakeDbConnectionPool.SetTimeout(10);`
- in the new pool using `SnowflakeDbConnectionPool.SetTimeout` is no longer possible, we should put it into the connection string
- I don't want to create if statements inside tests to differentiate between old and new pool to cover both cases differently. It is better to create to separate tests for each of the pool version

Reply:
As discussed it could be common with new connection pool switches (ignored by prev version) but for test clarity moving them to new place to differentiate by connection strings.